app/src/main/java/org/simple/clinic/sync/indicator/SyncIndicatorViewController.kt

```diff
@@ -65,25 +67,24 @@ class SyncIndicatorViewController @Inject constructor(
     return showSyncIndicatorState.mergeWith(showDefaultSyncIndicatorState)
   }
 
-  private fun syncedFailureState(timestamp: Instant?): SyncIndicatorState {
+  private fun syncedFailureState(timestamp: Instant?, maxIntervalSinceLastSync: Duration): SyncIndicatorState {
```

Comment:
Did you intentionally not rename `maxIntervalSinceLastSync` to `syncFailureThreshold?

Reply:
Its renamed in the later commit.